specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json

```diff
@@ -2305,13 +2321,14 @@
         "type": {
           "type": "string",
           "enum": [
-            "SystemAssigned"
+            "SystemAssigned",
+            "None"
           ],
           "x-ms-enum": {
             "name": "DiskEncryptionSetIdentityType",
             "modelAsString": true
           },
-          "description": "The type of Managed Identity used by the DiskEncryptionSet. Only SystemAssigned is supported."
+          "description": "The type of Managed Identity used by the DiskEncryptionSet. Only SystemAssigned is supported for new creations. Identity type None is supported on existing Disk Encryption Set for AAD Tenant migration."
```

Comment:
Last sentence. "Disk Encryption sets can be updated with Identity type None during migration of subscription to a new  Azure Active Directory tenant; it will cause the encrypted resources to lose access to the keys."